sub/go.mod

```diff
@@ -2,9 +2,10 @@ module github.com/okteto/pipeline-with-terraform/sub
 
 go 1.17
```

Comment:
@irespaldiza Should we update this project to go 1.18? Just mentioning because we are just upgrading the Go version on the Dockerfile

cc @odedlaz

Reply:
Yes @jLopezbarb

Reply:
@rberrelleza , anything I can help with?